mmv1/third_party/terraform/services/sql/resource_sql_database_instance.go.tmpl

```diff
@@ -2148,6 +2149,16 @@ func maintenanceVersionDiffSuppress(_, old, new string, _ *schema.ResourceData)
 	}
 }
 
+func databaseVersionDiffSuppress(_, old, new string, _ *schema.ResourceData) bool {
+	// Ignore the minor version in database version diff for MySQL 8.0.
+	if strings.Contains(old, "MYSQL_8_0") && strings.Contains(new, "MYSQL_8_0") {
```

Comment:
This needs more complex logic to actually check the patch version. As written, it will prevent manual upgrades from (for example) MYSQL_8_0_18 to MYSQL_8_0_26.

Reply:
Done